api/envoy/api/v2/cds.proto

```diff
@@ -804,6 +804,11 @@ message Cluster {
   // maybe by allowing LRS to go on the ADS stream, or maybe by moving some of the negotiation
   // from the LRS stream here.]
   core.ConfigSource lrs_server = 42;
+
+  // If track_timeout_budgets is true, the timeout budget histograms will be published for each
```

Comment:
Can you describe which stats are emitted and ref-link to those stats. Also describe the timeout precision comment so the user understands what is emitted.